### ISSUE TITLE
Milestone 2 feedback

### ISSUE THREAD

**Issue by andytai7** [UBC-MDS/canadian_heritage_funding#63]
Dear group 13,

You guys have made improvements! Great job! Below highlight some feedback for milestone 2. In addition, things under Suggestions: should be taken as suggestions and do not affect your mark.

**Expectations: Mechanics**

Comments
Xiangwxt only had one commit. Please spread the commits around to the other team members. 

**Project organization and documentation expectations: Mechanics**

Comments
Instead of just keeping one, there are two readmes in the root director.

**Project organization and documentation expectations: Reasoning**

Comments
Suggestion:
For EDA in the report, i would suggest adding a bit more such as a heatmap.

I would also suggest adding more to the metrics for measuring model performance.

In summary, i think you guys are doing a great job!

**Comment by aimee0317** [comment]
Hi Andy, thanks for the insightful feedback! Our group will make progress on our project accordingly.
